lightning/src/ln/channelmanager.rs

```diff
@@ -4865,6 +4874,53 @@ mod tests {
 			}
 		}
 	}
+
+	#[test]
+	fn test_notify_limits() {
```

Comment:
Weirdly when I run this test, only `trace` logs print out. Here's my log code: https://i.imgur.com/W37yQmk.png and here's what's printing out: https://i.imgur.com/DhpuX7u.png. Can't repro from randomly selected tests on `main`, but haven't investigated further

Reply:
See second commit at #976